### PR TITLE
[DOCS] Update README files

### PR DESCRIPTION
**Pull request by Krlier**
# Closes issue #63

### CODE REVIEW

File: owasp-top10-2017-apps/a2/saidajaula-monster/README.md [hunk 1]
```diff
@@ -27,16 +27,16 @@ Then simply visit [localhost:10082][App] !
 
 ## Attack Narrative
 
-To understand how this vulnerability can be exploited, check [this] other section!
+To understand how this vulnerability can be exploited, check [this section]!
```
Comment (rafaveira3) [inline owasp-top10-2017-apps/a2/saidajaula-monster/README.md]: Suggestion: What do you think of `[this section](docs/ATTACK.md)`?

File: owasp-top10-2017-apps/a3/insecure-go-project/README.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ WORKING!
 
 ## Attack Narrative
 
-To understand how this vulnerability can be exploited, check this section!
+To understand how this vulnerability can be exploited, check [this section]!
```
Comment (rafaveira3) [inline owasp-top10-2017-apps/a3/insecure-go-project/README.md]: Suggestion: What do you think of `[this section](docs/ATTACK.md)`?

File: owasp-top10-2017-apps/a5/ecommerce-api/README.md [hunk 3]
```diff
@@ -49,7 +49,7 @@ Hey, rafael! This is your ticket: rafael-08e8805f-422b-477b-8565-b0876b89da17
 
 ## Attack Narrative
 
-To understand how this vulnerability can be exploited, check [this] section!
+To understand how this vulnerability can be exploited, check [this section]!
```
Comment (rafaveira3) [inline owasp-top10-2017-apps/a5/ecommerce-api/README.md]: Suggestion: What do you think of `[this section](docs/ATTACK.md)`?

File: owasp-top10-2017-apps/a8/amarelo-designs/README.md [hunk 4]
```diff
@@ -25,7 +25,7 @@ Then simply visit [localhost:5000][App] !
 
 ## Attack Narrative
 
-To understand how this vulnerability can be exploited, check [this] other section!
+To understand how this vulnerability can be exploited, check [this section]!
```
Comment (rafaveira3) [inline owasp-top10-2017-apps/a8/amarelo-designs/README.md]: Suggestion: What do you think of `[this section](docs/ATTACK.md)`?

File: owasp-top10-2017-apps/a9/cimentech/README.md [hunk 5]
```diff
@@ -27,7 +27,7 @@ Then simply visit [localhost:80/][App] !
 
 ## Attack Narrative
 
-To understand how this vulnerability can be exploited, check this [other][Narrative] section!
+To understand how this vulnerability can be exploited, check [this section]!
```
Comment (rafaveira3) [inline owasp-top10-2017-apps/a9/cimentech/README.md]: Suggestion: What do you think of `[this section](docs/ATTACK.md)`?

File: owasp-top10-2017-apps/a2/saidajaula-monster/README.md (hunk 1 above)
Comment (Krlier) [reply owasp-top10-2017-apps/a2/saidajaula-monster/README.md]: Thanks @rafaveira3 ! But the link to `[this section]` is at the end of the file!

File: owasp-top10-2017-apps/a3/insecure-go-project/README.md (hunk 2 above)
Comment (Krlier) [reply owasp-top10-2017-apps/a3/insecure-go-project/README.md]: Thanks @rafaveira3 ! But the link to `[this section]` is at the end of the file!

File: owasp-top10-2017-apps/a5/ecommerce-api/README.md (hunk 3 above)
Comment (Krlier) [reply owasp-top10-2017-apps/a5/ecommerce-api/README.md]: Thanks @rafaveira3 ! But the link to `[this section]` is at the end of the file!

File: owasp-top10-2017-apps/a8/amarelo-designs/README.md (hunk 4 above)
Comment (Krlier) [reply owasp-top10-2017-apps/a8/amarelo-designs/README.md]: Thanks @rafaveira3 ! But the link to `[this section]` is at the end of the file!

File: owasp-top10-2017-apps/a9/cimentech/README.md (hunk 5 above)
Comment (Krlier) [reply owasp-top10-2017-apps/a9/cimentech/README.md]: Thanks @rafaveira3 ! But the link to `[this section]` is at the end of the file!

**Pull request merged**